### DISCUSSION TITLE
Getting Started with LabOp

### DISCUSSION THREAD

**Discussion by vancedave** [Bioprotocols/labop#200] [Q&A]
Hello! Trying to get started with Lab Op. When I attempt to install LabOp I get this error. Any thoughts?
<img width="506" alt="Error" src="https://github.com/Bioprotocols/labop/assets/123413523/fc896b01-a63e-4432-81fa-3682b36441d3">

**Comment by jakebeal** [comment]
Looks like this is happening because the documentation is wrong: labop hasn't been published to PyPi, so you'll need to install locally. 
cd into labop, then run `pip3 install .`

I've filed #201 regarding the bad documentation.
